.github/workflows/deploy.yml

```diff
@@ -183,7 +183,7 @@ jobs:
 
       - name: Prepare Image Tags
         run: |
-          echo "TAG_REPEATER=neuralegion/repeater" >> $GITHUB_ENV
+          echo "TAG_REPEATER=brightsec/repeater" >> $GITHUB_ENV
```

Comment:
Let's undo this line of code until the repository is created in Docker Hub. Please open a subtask under the relevant ticket in Jira